.circleci/config.yml

```diff
@@ -124,7 +124,7 @@ workflows:
             - lint
           matrix:
             parameters:
-              ruby-version: ["2.2", "2.3", "2.4", "2.5", "2.6", "2.7"]
+              ruby-version: ["2.2", "2.3", "2.4", "2.5", "2.6", "2.7", "3.0"]
```

Comment:
Note: Ruby 2.2 is included in the test matrix and is [the minimum version declared in the gemspec](https://github.com/honeycombio/beeline-ruby/blob/v2.7.0/honeycomb-beeline.gemspec#L18), but [the README says Ruby 2.3+ is required](https://github.com/honeycombio/beeline-ruby/blame/v2.7.0/README.md#L16). I'm inclined to let it ride for now.